Add extender docs (#933)
radius-project/docs: docs/content/guides/author-apps/custom/overview/index.md

---
type: docs
title: "Overview: Custom resource types"
linkTitle: "Overview"
description: "Learn how to model and deploy your own resource types in Radius"
weight: 100
---

> When the types built into Radius aren't enough, you can model and deploy your own resource types. Currently, Radius supports an untyped "extender" resource which allows you to pass in any property or secret.

Reviewer: Suggest rewording to avoid saying that Radius is not enough :) 
```suggestion
When you have a need for types beyond those currently built into Radius, you can model and deploy your own resource types. Currently, Radius supports an untyped "extender" resource which allows you to pass in any property or secret.
```
